dsc/tests/dsc_args.tests.ps1

```diff
@@ -61,7 +61,7 @@ Describe 'config argument tests' {
 '@ }
     ) {
         param($text)
-        $output = $text | dsc resource get -r *registry
```

Comment:
We should have a specific case for wildcard searching if you're replacing these

Reply:
With this change `get/set/test` operations no longer support wildcards. only `List` does.
I had to do this to simplify current changes.
I've created issue #247 to track this if we need to get this back.